systools/serial_log_decoder/serial_log_decoder.py

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/env python3
```

Comment:
May we update Licensing headers (both side)

Reply:
https://github.com/SiliconLabs/UnifySDK/pull/49

Please resolve if OK.